README.md

```diff
@@ -155,31 +170,57 @@ val qbeastTable = QbeastTable.forPath(spark, tmpDir)
 
 qbeastTable.getIndexMetrics()
 
-qbeastTable.analyze()
+```
+
+### 6. Optimize the table
+
+**Optimize** is a very expensive operation that consist on **rewriting part of the files to accomplish an even distribution of the records withing files**.
```

Comment:
Could we please explain what optimizing a table entails? We said yesterday that it is about moving offsets to their legitimate cube/file and performing rollups of small files into their parents..

Reply:
Do you think such specific information should be on the README? Or is it better to move all the explanation about optimize operation into the QbeastTable.md ?

Reply:
I've moved part of the definition and example to the `QbeastTable` doc.

Reply:
That's fine, maybe point here to the QbeastTable doc for completeness.

Reply:
Yep, I've added two links: one in the Revision paragraph and other at the end of the section 👍